### ISSUE TITLE
switch role to tree / treeitem 

### ISSUE THREAD

**Issue by pkra** [krautzource/mathjax-sre-walker#33]
It works fairly well and matches the ARIA semantics well.

**Comment by pkra** [comment]
Quick test in NVDA is quite good (only problem is that it always reads top-level label on focus, not the activedescendant when returning to an expression). JAWS is good (does not voice top-level label on focus but tree nav info, has the usual problems leaving focus), ORCA (CSS top-level label ignored and no focus).

**Comment by pkra** [comment]
I did not use group because that leads to the usual group problems. It would seem natural to use tree on children that have children. That would be equally well working except for tree depth and position calculations / voicing in NVDA (it voices the position on the level if the child with child has treeitem but not if it has tree). But that might be a general issue and might be fixable by adding posinset etc. Something to test later.
